.github/workflows/_rust.yml

```diff
@@ -64,6 +64,79 @@ jobs:
       - uses: ./.github/actions/setup-rust
       - run: cargo test --all-features ${{ matrix.packages }}
 
+  temp-build-windows-artifacts:
+    runs-on: windows-2019
+    defaults:
+      run:
+        working-directory: ./rust
+    strategy:
+      fail-fast: false
+      # The matrix is 1x1 to match the style of build-push-linux-release-artifacts
+      # In the future we could try to cross-compile aarch64-windows here.
+      matrix:
+        name:
+          - package: hello-world
+            artifact: hello-world
+    env:
+      BINARY_DEST_PATH: ${{ matrix.name.artifact }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/setup-rust
+        with:
+          targets: x86_64-pc-windows-msvc
+      - name: Build release binaries
+        run: |
+          cargo build --release -p ${{ matrix.name.package }}
+          cp target/release/${{ matrix.name.package }}.exe "${{ env.BINARY_DEST_PATH }}-x64.exe"
+          pwd
+          ls *.exe
+      - name: Save build artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ matrix.name.artifact }}-x64.exe
+          path: ${{ github.workspace }}/rust/${{ env.BINARY_DEST_PATH }}-x64.exe
+
+  # This should be identical to `build-push-windows-release-artifacts` in `cd.yml` except for the permissions, needs, and uploading step
+  temp-build-tauri:
+    runs-on: windows-2019
+    defaults:
+      run:
+        working-directory: ./rust
+    strategy:
+      fail-fast: false
+      # The matrix is 1x1 to match the style of build-push-linux-release-artifacts
+      # In the future we could try to cross-compile aarch64-windows here.
```

Comment:
👍

Reply:
Also the iteration cycle was so long that I feared if I brought back the architecture variable in the matrix, I'd be on it another 2 hours. Since aarch64 Windows hasn't really caught on yet, we could probably wait another few months for that